packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -50,13 +50,13 @@ export const pageQuery = graphql`
 
 <content>
 
-## Font files
+## Fonts
+
+Fonts in Paste are preloaded into Figma so you don't have to worry about downloading or installing them.
 
 ### Fira Mono
 
-If you are using <strong>any theme</strong> in Paste, you will need
-to [download and install Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono),
-our monospace typeface for code. It’s a free Google font.
+[Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono) is a free Google font and our monospace typeface for code. If you are using <strong>any theme</strong> in Paste, you will need to use it.
```

Comment:
Can we do a similar rewrite for all of the typefaces, to remove the language around needing to download and install it?

Reply:
oh nice! Didn't realize Google Fonts has the variable font now.

Reply:
Might sound a little less like people need to do something in order to use the fonts if the sentences are reworded like:

> { font name } is used in { x } theme in Paste.